microsoft/surface/common/kernel/default.nix

```diff
@@ -5,7 +5,7 @@ let
 
 in {
   imports = [
-    ./linux-6.6.x
+    ./linux-6.7.x
```

Comment:
If you're not going to expose the 6.6.x `.nix` files, you may as well delete them, yeah?

In the past, @Mic92 has asked people to try to minimise the number of different kernel versions that have to be maintained for any particular hardware.

Reply:
Yeah, sound like a good idea.